src/AcceptanceTests/StepDefinitions/ApprenticeshipCreatedStepDefinitions.cs

```diff
@@ -74,6 +73,9 @@ public async Task ThenAnApprovalCreatedEventIsPublished()
             publishedEvent.UKPRN.Should().Be(ApprenticeshipCreatedEvent.ProviderId);
             publishedEvent.Uln.Should().Be(ApprenticeshipCreatedEvent.Uln);
             publishedEvent.DateOfBirth.Should().Be(ApprenticeshipCreatedEvent.DateOfBirth);
+            publishedEvent.ActualStartDate.Should().Be(ApprenticeshipCreatedEvent.ActualStartDate);
+            publishedEvent.StartDate.Should().Be(ApprenticeshipCreatedEvent.StartDate);
```

Comment:
Can we rename StartDate to PlannedStartDate on the outbound event please?  In an ideal world we'd rename the event in das-commitments as well but that'd be too much work.